ProjectLighthouse.Servers.GameServer/Controllers/Slots/PublishController.cs

```diff
@@ -256,6 +283,8 @@ public async Task<IActionResult> Publish([FromQuery] string? game)
         if (usedSlots > user.EntitledSlots)
         {
             Logger.Warn("Rejecting level upload, too many published slots", LogArea.Publish);
+            await this.database.SendNotification(user.UserId,
+                "Level upload was rejected by the server because you have published too many slots.");
```

Comment:
```suggestion
            await this.database.SendNotification(user.UserId,
                "Your level failed to publish because you have reached the maximum number of levels on your earth.");
```
Just another example of how these could be simplified